src/fileformats/ocd_file_import.cpp

```diff
@@ -1131,6 +1140,39 @@ void OcdFileImport::importLayoutObjects(const QString& param_string)
 			if (ok)
 				object_number = i_value;
 			break;
+		case 'x':
+			f_value = param_value.toDouble(&ok);
+			if (ok)
+				x_value = f_value;
+			break;
+		case 'y':
+			f_value = param_value.toDouble(&ok);
+			if (ok)
+				y_value = f_value;
+			break;
+		case 'a':
+		case 'b':
```

Comment:
As a side note. As to my understanding, a is in principle the angle of image x axis and b is the angle of the y axis in the map grid. The TODO should not be addressed in this patch set but if someone wants to look into it, here's the clue.

Of course, it will be easier for the future coder to adjust just one place in the code rather than fixing templates and image objects separately.

Reply:
I started having a look on that topic and already have a test file. Like you I decided that this fix is subject of another PR and thus deferred it.

Reply:
I also spent some time with the topic. I can load and save template transformation in .ocd files (see the attached archive) but the change breaks all the other template handling in Mapper.
[template-ab-params.zip](https://github.com/OpenOrienteering/mapper/files/10919682/template-ab-params.zip)

The thing is that `TemplateTransform::template_shear` is currently an opaque number that somehow embodies the actual template shear, along with other values like scaling and rotation. That said, the patch that introduces this class member needs a thorough review.